### PR TITLE
Show badges on Validator Explorer

### PR DESCRIPTION
**Pull request by Pedro-vk**
### Description

_Show badges on Validator Explorer_

### Tested

_Tested on Chrome._

### CODE REVIEW

File: packages/web/src/dev/ValidatorsListBadges.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+const allBadges = [
+  { src: '/images/badges/1st-proposer.svg', list: lProposer },
+  { src: '/images/badges/1st-transaction.svg', list: lTransaction },
+  { src: '/images/badges/attestation.svg', list: lAttestation },
+  { src: '/images/badges/founder-validator.svg', list: lFounder },
+  { src: '/images/badges/genesis-validator.svg', list: lGenesis },
+  { src: '/images/badges/master-validator.svg', list: lMaster },
+  { src: '/images/badges/vote-recipient.svg', list: lVote },
+].map((_) => ({ ..._, list: (_.list as any).toLowerCase() as string }))
```
Comment (aaronmgdr) [inline packages/web/src/dev/ValidatorsListBadges.tsx]: please give this variable a name. `_` should only be used in cases where the variable is not used.

Comment (aaronmgdr) [reply packages/web/src/dev/ValidatorsListBadges.tsx]: Also I believe `list` can be transformed into a Set like 

`.map((addresses) => ({ ...addresses, list:  new Set(addresses.split('/n')) }))`

this way below you can do 

 `allBadges.filter(({ list }) => list.has(address.toLowerCase()))`

and avoid looping inside a loop while rendering.

File: packages/web/public/images/badges/vote-recipient.svg [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (aaronmgdr) [inline packages/web/public/images/badges/vote-recipient.svg:1]: I see we already have badges in another folder. can we deduplicate?

File: packages/web/src/dev/ValidatorsListBadges.tsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+interface Props {
+  address: string
+}
+
+class ValidatorsListBadges extends React.PureComponent<Props & I18nProps> {
+  state = {
```
Comment (aaronmgdr) [inline packages/web/src/dev/ValidatorsListBadges.tsx]: remove this state

**Review by aaronmgdr — CHANGES_REQUESTED** [review]
good work. just a few things to do

File: packages/web/src/dev/ValidatorsListBadges.tsx (hunk 1 above)
Comment (Pedro-vk) [reply packages/web/src/dev/ValidatorsListBadges.tsx]: `allBadges.filter(({ list }) => list.includes(address.toLowerCase()))`
This part of the code is not a double loop, is just a string matching. List is a long string with addresses (and new lines), so just checking if the string contains the address.

But anyway, I will change it.

File: packages/web/public/images/badges/vote-recipient.svg (hunk 2 above)
Comment (Pedro-vk) [reply packages/web/public/images/badges/vote-recipient.svg:1]: They are not the same, this badges are without borders.

**Review by aaronmgdr — APPROVED** [review]

**Pull request merged**